device/esp_tinyusb/Kconfig

```
@@ -285,6 +285,41 @@ menu "TinyUSB Stack"
             config TINYUSB_NET_MODE_NONE
                 bool "None"
         endchoice
+
+        config TINYUSB_NCM_OUT_NTB_BUFFS_COUNT
+            int "Number of NCM NTB buffers for reception side"
+            depends on TINYUSB_NET_MODE_NCM
+            default 2
+            range 1 4
+            help
+                Number of NTB buffers for reception side.
+                Can be increased to improve performance with the cost of additional RAM requirements.
+
+        config TINYUSB_NCM_IN_NTB_BUFFS_COUNT
+            int "Number of NCM NTB buffers for transmission side"
+            depends on TINYUSB_NET_MODE_NCM
+            default 2
+            range 1 4
+            help
+                Number of NTB buffers for transmission side.
+                Can be increased to improve performance with the cost of additional RAM requirements.
```

Maybe it is better to provide some details, why do we need to change this config. As we don't have any tool to measure the numbers so far, maybe we can provide only general idea: 

```suggestion
                Can be increased to improve performance with the cost of additional RAM requirements.
                Possible options: 
                1 - Good performance, but some packet could be lost
                2 - Better performance on small packets, but sometimes transfer request could be blocked with an error
                 3 - Better performance, transfer request can never be blocked
                 4 - No additional gain
```

---

I was not able to prove any of those statements, so I bit rephrased the description. But yes, I can put some guidelines there.

---

I didn't know that you have tested it. 
If these statements are not relevant, then there is no point to change anything.

---

> Good performance, but some packets could be lost

@roma-jam What do you mean by 'lost packets' here? The esp_netif will handle the packet retransmission, and as far as I know, this is considered normal in networking. 'lost packet' could scare some users about data being entirely lost...

---

@tore-espressif lost events.

Lost packets sounds scaring, indeed.